documentation/doxygen/Makefile

```diff
@@ -40,18 +41,22 @@ images:
 	cp images/* $(DOXYGEN_IMAGE_PATH)
 
 pyzdoc:
+	$(call MkDir,$(DOXYGEN_PYZDOC_PATH))
 	$(PYTHON_EXECUTABLE) extract_docstrings.py $(PYZ_DIR)
 	$(PYTHON_EXECUTABLE) print_roofit_pyz_doctrings.py > $(PYZ_DIR)/_roofit.pyzdoc
+	mv $(PYZ_DIR)/*.pyzdoc $(DOXYGEN_PYZDOC_PATH)
```

Comment:
I think it would be cleaner to pass to `extract_docstrings.py` and `print_roofit_pyz_doctrings.py` (I just noticed the missing `s` in the name of this script btw @guitargeek ) the output path (`$(DOXYGEN_PYZDOC_PATH)`) so they write directly there, instead of generating in `$(PYZ_DIR)` and then moving the results of both.

Reply:
I did it that way to avid to modify your code to much guys ...

Reply:
Well in the previous version `$(PYZ_DIR)` was both the input and the output directory, but now you created a new output directory, so it makes sense the python scripts write there directly. @guitargeek can perhaps comment too

Reply:
Yes and it has also the advantages to avoid the root sources polluting (there was left over)